### PR TITLE
Swap out keys so they use mine

### PR DESCRIPTION
**Pull request by isiahmeadows**
**Edit:** Actually fill out the template

## Description

Cycling the keys that weren't mine to ones that are.

## Motivation and Context

This is upon @tivac's request, particularly for the release API key.

## How Has This Been Tested?

That's what the Travis run is for. If this screws up somehow during release, I'll figure it out then anyways, so I can patch it then.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation change

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated `docs/change-log.md` **N/A**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -75,20 +75,20 @@ after_success:
 env:
   global:
     # Set up GH_USER_EMAIL & GH_USER_NAME env variables used by travis-scripts package
-    - secure: Xvqvm3+PvJu/rs3jl/NNn0RWLkkLkIoPHiL0GCfVRaywgjCYVN02g54NVvIDaOfybqPmu9E6PJFVs92vhF34NMFQHf4EWskynusIGV271R2BV0i+OJBfLMuLgiwm6zRn7/Zw4JvWIUGEwcnlz0qxbqdHsS0SOR3fIkFzePickW0=
-    - secure: Rf/ldEO9d4vItJhe6EmqWpFAyCARzoCb422nHnjr1hYJknnwIXpgyZ1C/7On/9o7rWPPf+8WcHC/rgjK2rthKCldzdG5I60LfWSNzap9lk3Aa4TpSCoDBuEp7JVvDr5tc3rKnBXVT71hOay7RSx1StWzXiJs9mjaeVMJzYzRT78=
+    - secure: UdSk2uKTL56iOHkIZP9Tpj/OI8w26DRTs6INRq+peZKkHq8NVC0TmbtbpRoc5kxErovN2hyqsAnoMtXSQHKK+H9mpMx0v4ck/I+o2oEny/1hwi5YQ/Q0nAebVhZkgA3eVhJY1brK0bOlr8uI07m9mcPs3Qz0ramZutJQG7FdnZs=
```
Comment (isiahmeadows) [inline .travis.yml]: When I created both via a single command, it resulted in a single secure string, not two.

File: .travis.yml [hunk 2]
```diff
@@ -75,20 +75,20 @@ after_success:
...
     api_key:
-      secure: PauFuz+pn7oRpHn2JTl4k3+iWjOofyBYBvavPQVNdXgKws9mGj0i2n5k2oIDU09VD7NeyEkwP6tdLCUFNaR8uwTJH/TBXMZE95oxUEaliFreA0nOiI3WkG4NCW0GwUoIIn1yL14y6+9oEBinWUia8DIn9kZNS11DNDgQpIPnoQQ=
+      secure: BBlVwr3CRWS6Zmc8nsYMMN59P95g/lg9OzNCsP7uiFyu7mnC6VdOIgwmmJXBviUiEymbcdepgRqEdjHIoemg2YhM5IOnhMoYFrfMBoWUpMihpejFY8EVm3NrTh0prXgCHbp/3OktoKdOn/Zhc2z7cKDMeToHzaDStLtQPR8u9jU=
     file:
       - "mithril.js"
       - "mithril.min.js"
       - "mithril.mjs"
       - "mithril.min.mjs"
     skip_cleanup: true
+    draft: true
```
Comment (isiahmeadows) [inline .travis.yml]: Drive-by, so I have time to edit it *before* the release goes live.

**Comment by tivac** [comment]
Looks reasonable to me, unfortunately short of making a new testing repo this stuff is tricky to actually verify :-\

**Comment by isiahmeadows** [comment]
@tivac Thanks for taking a quick look. I was just asking for a sanity check to make sure I didn't do anything *clearly* wrong here. I expect if I screwed anything up, I'll figure that out pretty quickly while trying to release. 😄
